rcgen/src/lib.rs

```diff
@@ -158,6 +158,14 @@ impl<'a, S: SigningKey> Issuer<'a, S> {
 		}
 	}
 
+	/// Create a new issuer fromthe given parameters and signing key references.
+	///
+	/// Use [`Issuer::new`] instead if you want to obtain an [`Issuer`] with a static
+	/// lifetime.
+	pub fn from_refs(params: &'a CertificateParams, signing_key: &'a S) -> Self {
```

Comment:
I think it could make sense to expose this, but I don't think it's useful to wrap `from_params()` when we can just expose it directly? Also wondering about the best name, I don't like `from_refs()` much, I think `from_params()` is probably okay.